### DISCUSSION TITLE
能否取消倍速记忆的弹窗。

### DISCUSSION THREAD

**Discussion by knItalian** [the1812/Bilibili-Evolved#3253] [功能建议 / Ideas]
每次都弹 ,看播放器就知道了 没必要再弹出一次 。
希望取消掉这个。

![截图20220417141417](https://user-images.githubusercontent.com/32791144/163703266-7c4baea8-6504-407c-a115-cc0488c1f63f.png)

**Comment by LonelySteve** [comment]
![image](https://user-images.githubusercontent.com/34429322/163703386-af46a0e0-6c8b-4668-9f42-12e3c8893cc9.png)
